evaluate_library_access_from_output_tsv/create_stratefied_sample_of_dois.R

```diff
@@ -0,0 +1,72 @@
+#### Load libraries ####
+
+library('dplyr')
```

Comment:
To ensure that all namespaces are declared, while still using pipes, you can do:

```R
# Load magrittr pipe
`%>%` = dplyr::`%>%`
```

In other words, I try never to call library in R and instead just always use `::`

Reply:
That's fine with me. I've implemented this in 04a29768.
Similarly, I added several `R` dependencies to `environment.yml` in bfda19c2.